src/components/common/UserAvatar/UserAvatar.tsx

```diff
@@ -0,0 +1,17 @@
+import styles from './userAvatar.module.scss';
+
+type UserAvatarProps = {
+    userProfileImageUrl: string;
+};
+
+const UserAvatar = ({ userProfileImageUrl }: UserAvatarProps) => {
+    return (
+        <img
+            src={userProfileImageUrl}
+            alt="Avatar"
+            className={styles['user-avatar']}
+        />
+    );
+};
+
+export default UserAvatar;
```

Comment:
Can we use named exports instead of default exports in all new components, as this makes it easy to find references?